fix: fix and refactor gex app (#81)
ignite/apps: explorer/CHANGELOG.md

# Explorer App Changelog

## [`v0.3.0`](https://github.com/ignite/apps/releases/tag/explorer/v0.3.0)

> * The Gex app was not working when we passed a custom RPC address.

Reviewer: ```suggestion
* The Gex app was malfunctioning when passing a custom RPC address.
```
